vg/build/Dockerfile

```diff
@@ -1,24 +1,25 @@
-FROM ubuntu:14.04
+FROM ubuntu:15.10
```

Comment:
It's actually this change which fixes the gcc-4.9.
I'll fixup the other extraneous stuff.